internal/virtualKubelet/node/env.go

```diff
@@ -324,24 +240,29 @@ loop:
 }
 
 // makeEnvironmentMap returns a map representing the resolved environment of the specified container after being populated from the entries in the ".env" and ".envFrom" field.
-func makeEnvironmentMap(ctx context.Context, pod *corev1.Pod, container *corev1.Container, namespaceMapper namespacesMapping.NamespaceNatter, apiController controller.ApiControllerCacheManager, rm *manager.ResourceManager, recorder record.EventRecorder, res map[string]string) error {
+func makeEnvironmentMap(ctx context.Context, pod *corev1.Pod, container *corev1.Container, rm *manager.ResourceManager, recorder record.EventRecorder, res map[string]string) error {
 
 	// TODO If pod.Spec.EnableServiceLinks is nil then fail as per 1.14 kubelet.
-	enableServiceLinks := corev1.DefaultEnableServiceLinks
-	if pod.Spec.EnableServiceLinks != nil {
-		enableServiceLinks = *pod.Spec.EnableServiceLinks
-	}
+	/*
```

Comment:
Can we remove this commented code?

Reply:
It is part of the commit from @aleoli we reverted (original virtual kubelet code). I commented it to restore the previous status of that code. Since we have it in the story I could also delete it.